doc/apps/sceewlog.rst

```diff
@@ -119,28 +118,28 @@ in :confval:`activeMQ.bnaFile`.
 
 .. code-block:: sh
 
-   activeMQ.global.bnaPolygonName = none
-   activeMQ.America.bnaPolygonName = America
+   activeMQ.profile.global.bnaPolygonName = none
```

Comment:
Oups, only the example was wrong? Not the default config?

Reply:
Yes, It was changed some time ago within the config file